tests/queries/0_stateless/02415_all_new_functions_must_be_documented.reference

```diff
@@ -530,6 +530,10 @@ parseDateTimeBestEffortUS
 parseDateTimeBestEffortUSOrNull
 parseDateTimeBestEffortUSOrZero
 parseDateTimeInJodaSyntax
+parseDateTimeInJodaSyntaxOrNull
```

Comment:
Shouldn't we add documentation for these functions instead of adding it to the exceptions?

Reply:
In general, I agree with you. The problem is that the official documentation is currently not generated from the in-source documentation. This means both can easily get out-of-sync, and I like to avoid that. Actually most functions "choose" this (unsatisfying) approach. The real fix would be to have a CI job which generates the official docs from the in-source docs.